### ISSUE TITLE
Can't save anything in Zyn-Fusion.  Missing filename input box in UI

### ISSUE THREAD

**Issue by jdfight** [zynaddsubfx/zyn-fusion-issues#142]
Installed zynaddsubfx-git from kxstudio repository.  Things were going great until I tried to save an instrument.  I am unable to save any patches in zyn-fusion.  I can't type a filename anywhere in the UI.  I tried typing in the textbox to the left of the enter button at the top,  but nothing happens when I type there.   If I select a folder and hit "Enter" I am left with a file that has no name,  just a .xiz extension.  If I rename this file and try to load it in zyn-fusion,  It does not load any of the settings that I attempted to save. I get the same behavior when trying to save the master settings too.

I can successfully load settings saved from the older version of zynaddsubfx.

![zynscreen](https://user-images.githubusercontent.com/1063108/37260634-f76843dc-256b-11e8-8d67-72a8107f9a7d.jpg)

I am running System 76 Pop!_OS 17.10 (Ubuntu 17.10)
Memory: 23.5gb
Processor: Intel® Core™ i7-7700K CPU @ 4.20GHz × 8 
Graphics: GeForce GTX 1060 3GB/PCIe/SSE2
GNOME: 3.26.2
OS: 64-bit

**Comment by fundamental** [comment]
Can you test with the standalone (if you're using the plugin)? That should eliminate one variable in this issue.

**Comment by jdfight** [comment]
Yes, I have tried both in the plugin and standalone - same issue.

**Comment by fundamental** [comment]
@falkTX is there an easy way for me to know what day the zynaddsubfx-git package was built/rebuilt? I should likely know how to find that information for this and future bugs.

**Comment by fundamental** [comment]
I cannot replicate the issue on my machine. Are you able to type elsewhere in the interface? Does the path have any special characters?

**Comment by jdfight** [comment]
Okay - I don't know what I was doing before. I can type the filenames just
fine now.  I think I may have been confused by a lack of a cursor in the
text input box.  I did not realize that I could just start typing - I kept
trying to click the the input box expecting some sort of text entry mode.
This was the first time I have used the new fusion UI.  The "blank"
filenames were being created because I clicked the "enter" button expecting
some sort of file dialogue to appear.  But, when you click the enter button
before typing a name,  it just saves a blank filename and closes without
giving any confirmation. This could cause some headaches for end users who
are as unobservant as myself.
